lib/dal/tests/integration_test/attribute_value/subscription.rs

```diff
@@ -304,6 +304,89 @@ async fn subscribe_to_two_values(ctx: &mut DalContext) -> Result<()> {
     Ok(())
 }
 
+#[test]
```

Comment:
This test fails without the correction

Reply:
Props to you for a regression test, A++!